packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -725,11 +736,12 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             }
         };
 
-        const [chunks, metadata, electedSummarizerData, aliases] = await Promise.all([
+        const [chunks, metadata, electedSummarizerData, aliases, idCompressor] = await Promise.all([
             tryFetchBlob<[string, string[]][]>(chunksBlobName),
             tryFetchBlob<IContainerRuntimeMetadata>(metadataBlobName),
             tryFetchBlob<ISerializedElection>(electedSummarizerBlobName),
             tryFetchBlob<[string, string][]>(aliasBlobName),
+            tryFetchBlob<any>(".idCompressor"),
```

Comment:
extract string to constant to align with others?

Reply:
Extracted this to a const and am now using it